Format /learn using Prettier (#24048)
mdn/content: files/en-us/learn/forms/how_to_structure_a_web_form/index.md

> 3. Next, create your form by adding the outer {{htmlelement("form")}} opening tag:

Reviewer: ```suggestion
3. Next, create your form by adding a {{htmlelement("form")}} element:
```

   ```html
   <form></form>
   ```

> 4. Inside the `<form>` tags, add a heading and paragraph to inform users how required fields are marked:

Reviewer: ```suggestion
4. Inside the `<form>` element, add a heading and paragraph to inform users how required fields are marked:
```
